### DISCUSSION TITLE
Delete File Uploaded via HTTP Manually

### DISCUSSION THREAD

**Discussion by stefanLeo** [processone/ejabberd#3650] [Questions] [answered]
Hi 

I have a question in regards to XEP-0363 and mod_http_upload. 
Is there a way to manually delete uploaded files?  I would like to build my own retention policy directly as part of my XMPP client (e.g. auto-delete x seconds after upload if the user decides to do so) and also give users the chance to delete their files upon request. 

Does eJabberD support something like that already (e.g. via HTTP DELETE METHOD)?
If not - can this be added via a feature request? 

THX & BR
Stefan

**Comment by badlop** [comment, marked as answer]
Such a thing is not implemented. And XEP-0363 doesn't mention it, either.
